components/feature/findinpage/src/main/java/mozilla/components/feature/findinpage/FindInPageFeature.kt

```diff
@@ -19,10 +20,11 @@ import mozilla.components.support.base.feature.LifecycleAwareFeature
 class FindInPageFeature(
     sessionManager: SessionManager,
     view: FindInPageView,
+    engineView: EngineView,
```

Comment:
I think this a breaking API change, and we should commenting it out in the changelog :)
In the changelog file there are some examples of previous breaking API change.

![image](https://user-images.githubusercontent.com/773158/54608412-fddf6700-4a26-11e9-8553-fa3149d0fe3e.png)

Reply:
Done!

Reply:
👍